modules/developers-guide/examples/result/gql_1createDeploy.result

```diff
@@ -0,0 +1,10 @@
+{
+  "data": {
+    "deploySchema": {
+      "version": "4adc2e30-9e53-11eb-8fde-b341b9f82ca9",
```

Comment:
Probably don't want to diff this file in the test since this will always change.

Reply:
I'll leave in the example for displaying what a result looks like, but the test shouldn't test against it.